.github/workflows/workflow.yaml

```diff
@@ -0,0 +1,35 @@
+name: build_test
+on:
+  push:
+    paths-ignore:
+      - "**.md"
+      - "hack/**"
+  pull_request:
+    paths-ignore:
+      - "**.md"
+      - "hack/**"
+
+jobs:
+  lint:
+     name: "Lint"
+     runs-on: ubuntu-latest
+     steps:
+       - uses: actions/checkout@v2
+       - name: golangci-lint
+         uses: golangci/golangci-lint-action@v2
+         with:
+           version: v1.40.1
+
+#  test:
+#     name: "Unit Tests"
+#     runs-on: ubuntu-latest
+#     steps:
+#       - name: Set up Go 1.16
+#         uses: actions/setup-go@v2
+#         with:
+#           go-version: 1.16
+
+#       - name: Check out code into the Go module directory
+#         uses: actions/checkout@v2
+
+#       - name: Unit test
```

Comment:
why is this commented?

Reply:
we don't have unit tests yet, right?

Reply:
that's okay. it'll just pick it up once we have it. let's uncomment and add that